holidays/countries/barbados.py

```diff
@@ -0,0 +1,70 @@
+from holidays.calendars.gregorian import JAN, APR, JUL, AUG, NOV, DEC
+from holidays.holiday_base import HolidayBase
+from holidays.holiday_groups import ChristianHolidays, InternationalHolidays
+
+
+class Barbados(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """
+    https://www.timeanddate.com/holidays/barbados/
+    """
+
+    country = "BB"
+    default_language = "en"
```

Comment:
All right, so this is the reason why `make l10n` wasn't working. You set the `default_language` but didn't do any l10n work. As a quick fix you need to remove the `default_language` line.

Reply:
This might be a Windows specific issue. @KJhellico any suggestions here?

Reply:
I'm actually on a mac @arkid15r.